aptos-move/block-executor/src/executor.rs

```diff
@@ -506,13 +529,43 @@ where
         patched_resource_write_set
     }
 
+    fn map_id_to_values_in_read_set_sequential(
+        delayed_field_keys: Option<impl Iterator<Item = T::Identifier>>,
+        write_set_keys: HashSet<T::Key>,
+        read_set: RefCell<HashMap<T::Key, HashSet<T::Identifier>>>,
+        latest_view: &LatestView<T, S, X>,
```

Comment:
can you also pass unsync_map, so you don't need to add new methods to latest_view, just for the unsync_map? (i.e. fetch_from_unsync_map)

Reply:
Done.